extension/src/pipeline/index.ts

```diff
@@ -177,6 +153,29 @@ export class Pipeline extends DeferredDisposable {
     return appendFileSync(join(this.dvcRoot, TEMP_DAG_FILE), '\n')
   }
 
+  private findPipelineCwd() {
+    const focusedPipeline = this.getFocusedPipeline()
+    if (focusedPipeline) {
+      return focusedPipeline
+    }
+
+    const pipelines = this.model.getPipelines()
+    if (!pipelines?.size) {
```

Comment:
Out of scope, but since we have a lint rule to test zero length explicitly (`something.length === 0` instead of `!something.length`). Should we have a similar rule for `size` as well?

Reply:
Makes sense to me. I can look into a eslint rule for it!

Reply:
Looks like the lint rule we have (`explicit-link-check`) works for both arrays and size. This is allowed though since we have an undefined check there as well.